test-cases/performance/perf-regression-mixed-cdc.yaml

```diff
@@ -0,0 +1,43 @@
+test_duration: 30
+
+# stress_cmd_w: "cassandra-stress user profile=/tmp/cs_cdc_simple_write.yaml ops'(write=1)' no-warmup cl=QUORUM duration=15m -port jmx=6868 -mode cql3 native -rate threads=100"
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=15m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -pop seq=1..3000000"
+# stress_cmd_m: "cassandra-stress user profile=/tmp/cs_cdc_simple_write.yaml ops'(write=1, read_cdc_log)' no-warmup cl=QUORUM duration=15m -port jmx=6868 -mode cql3 native -rate threads=100"
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+
+instance_type_loader: 'c4.2xlarge'
+instance_type_monitor: 't3.small'
+instance_type_db: 'i3.4xlarge'
+
+user_prefix: 'staging-perf-cdc'
```

Comment:
The PR is for official job in scylla-master, so let's remove the staging-

Reply:
Mixed is not yet ready.

Reply:
file removed